src/applications/personalization/dashboard/containers/DashboardApp.jsx

```diff
@@ -399,6 +429,7 @@ export const mapStateToProps = state => {
     canAccessAppeals,
     canAccessClaims,
     profile: profileState,
+    showProfile2,
```

Comment:
It is too late to propose renaming Profile 2 to `ProfileEXTREME`?

Reply:
Never too late! 😄